src/Symfony/Component/Form/FormRendererEngineInterface.php

```diff
@@ -128,7 +128,6 @@ public function getResourceHierarchyLevel(FormView $view, array $blockNameHierar
      *
      * @param FormView $view      The view to render
      * @param mixed    $resource  The renderer resource
-     * @param string   $blockName The name of the block to render
```

Comment:
Why did you remove this one but not the others?

Reply:
Think it happened when rebasing. I added `string` declaration myself and then someone added it too without removing the phpdoc, so only phpdoc removal left here.
Should I remove `$view` in another PR targeting `3.4`? Or just revert my change in this class?